src/main/resources/com/sysdig/jenkins/plugins/sysdig/SysdigIaCScanBuilder/config.properties

```diff
@@ -0,0 +1,4 @@
+ListUnsupported=list-unsupported-resources
+ListUnsupportedDescr=Toggle output of detailed list of unsupported resources
+sysdigEnv=Sysdig Secure Engine URL
```

Comment:
```suggestion
sysdigEnv=Sysdig Secure Endpoint
```

Reply:
thats the same trminology used by the rest of cli